stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md

```diff
@@ -24,14 +24,14 @@ endpoint jdbc:Client testDB {
     dbOptions: { useSSL: false }
 };
 ```
-The full list of endpoint properties can be found in the `sql:PoolOptions` type.
+The full list of endpoint properties can be found listed under the `sql:PoolOptions` type located in the `types.bal` file of the SQL package directory.
```

Comment:
Don't we need a comma after "type" here?

Reply:
No, unless you rewrite it like this: "...under the `sql:PoolOptions` type, which is located in the `types.bal` file..."

Reply:
I see, thanks. I felt your version is easier to read, than a long one sentence that doesn't break anywhere.